context/app/static/js/components/workspaces/WorkspaceLaunchSuccessToast.tsx

```diff
@@ -3,16 +3,27 @@ import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 
-function WorkspaceLaunchSuccessToast(id: number) {
+interface WorkspaceLaunchSuccessToastProps {
+  id: number;
+  workspaceLaunched: boolean;
+}
+
+function WorkspaceLaunchSuccessToast({ id, workspaceLaunched }: WorkspaceLaunchSuccessToastProps) {
```

Comment:
I think I'd just prefer two functions `WorkspaceLaunchSuccessToast` and `WorkspaceCreateSuccessToast` in a central `WorkspaceToasts` file rather than depending on the `workspaceLaunchedProp`.

@NickAkhmetov what do you think?

Reply:
I agree, this would also tie in with https://hms-dbmi.atlassian.net/browse/CAT-49

Reply:
Agreed! Pushed an update with this + the remaining workspace toasts in a central file.